src/common/ipc/cluster.ipc.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * This channel is broadcast on whenever the cluster fails to list namespaces
+ * during a refresh and no `accessibleNamespaces` have been set.
+ */
+export const ClusterListNamespaceForbiddenChannel = "cluster:list-namespace-forbidden";
+
+export type ListNamespaceForbiddenArgs = [clusterId: string];
+
+export function argArgsListNamespaceFordiddenArgs(args: unknown[]): args is ListNamespaceForbiddenArgs {
```

Comment:
I understand it is some form of convention for naming such functions, but there're too many `args` in this name? I believe function name should be easy to pronounce. How about `hasForbiddenArgs`?

Reply:
lol, sure I meant to call this `are...` but yeah it is a mouthful. Your name is better.